bin/atat-web-api.ts

```diff
@@ -31,4 +32,8 @@ if (utils.isPossibleTemporaryEnvironment(environmentId)) {
 // cdk.Aspects.of(app).add(new NIST80053R5Checks({ verbose: true }));
 cdk.Aspects.of(app).add(new GovCloudCompatibilityAspect());
 
-const apiStack = new AtatWebApiStack(app, `${environmentName}AtatWebApiStack`, { environmentName });
+const apiStack = new AtatPipelineStack(app, `Pipeline`, {
+  environmentName,
+  branch: "feature/cdk-pipelines",
+  githubPatSecretPath: "auth/github/pat-7176", // pragma: allowlist secret
```

Comment:
This name is temporary. Ideally, we'd use something that doesn't anger the secret scanner. I want to remove the `// pragma: allowlist secret` because it sets a bad precedent.